client/src/components/NearestHospital/LocalEmergencyNo.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import styled from "styled-components";
+import { NoticeBar } from 'antd-mobile';
+
+import { theme, mq } from "../../constants/theme";
+
+
+const { colors } = theme;
+const { typography } = theme;
+const { xlarge } = typography.size;
+const { royalBlue } = colors;
+
+
+const LocalEmergencyNo = props => {
+
+
+  const EmergencyLine = styled(NoticeBar)`
+       height: auto;
+       background-color: #D1222C;
+       color: #fff;
+       h4 {
+         color: #fff;
+         margin-bottom: 0;
+       }
+       h3 {
+         color: #fff
+       }
+      @media screen and (min-width: ${mq.phone.wide.maxWidth}) {
+
+          border: 0.2px solid rgba(185, 185, 185, 0.5);
+          padding: 2rem;
+          padding-left: 4rem;
+          background-color: #fff;
+          height: auto;
+          .am-notice-bar-operation {
+            display: none;
+          }
+          h4 {
+            font-weight: bold;
+            color: #000;
+          }
+          h3 {
+            font-weight: bold;
+            color: ${royalBlue};
+            text-decoration: underline;
+            font-size: ${xlarge};
+          }
+
+        }
+
+        @media screen and (max-width: ${mq.tablet.wide.maxWidth}) {
+          padding: 0;
+          padding-left: 0;
+        }
+
+  `;
+
+
+  return (
+    <EmergencyLine mode="closable" icon={null}>
```

Comment:
On mobile it keeps showing when I switch tabs even after closing it

Reply:
Need to match designs on mobile
- fix position of the closing icon
- more padding needed
- need correct typography styles 
- need border-radius

current
![Screen Shot 2020-05-12 at 12 08 02 AM](https://user-images.githubusercontent.com/42228386/81649127-ad725800-93e4-11ea-89ab-8152aae5b8fa.png)

![Screen Shot 2020-05-12 at 12 07 51 AM](https://user-images.githubusercontent.com/42228386/81649128-ad725800-93e4-11ea-8b88-6125182beac1.png)